CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.1.0 (2020-06-15)
+
+- Updated modal heading to use HTML heading elements. [#95](https://github.com/blackbaud/skyux-modals/pull/95)
```

Comment:
Should one of those "headings" be "header?"

Reply:
`HTML heading` is the proper term for `h1`, `h2`, `h3` tags. However, the component we use for modal headers is `modal-header`, so yeah let's change that one.

Reply:
Done.